assets/tests/components/app.test.tsx

```diff
@@ -49,7 +50,7 @@ describe("App", () => {
   })
 
   test("shows data outage banner if there's a data outage", () => {
-    ;(useDataStatus as jest.Mock).mockImplementation(() => "outage")
+    ;(useDataStatus as jest.Mock).mockImplementationOnce(() => "outage")
```

Comment:
**suggestion**(non-blocking): what do you think of using `mockReturnValueOnce` here?